docs/source/configuration.rst

```diff
@@ -60,6 +63,29 @@ and, therefore, the actions are executed in that same order.
 
 See :ref:`Examples` for configuration file examples.
 
+Defaults
+~~~~~~~~
+
+- The value type of the ``defaults`` is a dictionary.
+
+This dictionary must have the following key:
+
+.. list-table::
+   :header-rows: 1
+   :widths: 1 1 2
+
+   * - Key Name
+     - Value Type
+     - Value Description
+   * - ``actions``
+     - dictionary of :ref:`Actions`
+     - A dictionary made of :ref:`Actions` that will be executed on the
+       matching pull requests.
+
+The ``defaults`` are  merged with ``pull_request_rules``.
+If options are already defined in ``pull_request_rules`` they are not overwrited by ``defaults``.
+See :ref:`Examples` for configuration file examples.
```

Comment:
```suggestion
The ``defaults`` section is used to define default configuration values for actions run by pull request rules and by :ref:`Commands`.

If the options are defined in ``pull_request_rules`` they are used, otherwise, the values set in ``defaults`` are used.
```